columnphysics/icepack_parameters.F90

```diff
@@ -646,12 +646,12 @@ subroutine icepack_init_parameters(   &
          TTTocn_in        ! for qsat over ocn
 
       character (len=*), intent(in), optional :: &
-         atmbndy_in ! atmo boundary method, 'default' ('ccsm3') or 'constant'
+         atmbndy_in   ! atmo boundary method, 'default' ('ccsm3') or 'constant'
```

Comment:
Sorry to be picky, would it be possible to not introduce whitespace changes ? It adds noise to the diff (even though we can hide it in the GitHub UI or with `git diff` options, it is still an additional step to take).